win/nuget/boost/Roaster.boost.v142.dyn.x64.targets

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemDefinitionGroup>
+    <ClCompile>
+      <AdditionalIncludeDirectories>$(MSBuildThisFileDirectory)..\lib\native\include\;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
+    </ClCompile>
+  </ItemDefinitionGroup>
+  <ItemGroup />
+</Project>
```

Comment:
<img width="229" alt="image" src="https://user-images.githubusercontent.com/5203025/113478657-c7190400-94bc-11eb-9b4c-a304e9d87556.png">
Recommend to add \n at the end.
Per unix standard, text file should end with \n.
Usually Linux editor (e.g. vim) does this automatically for you upon opening, which can cause modification in unrelated PR in the future.
On the other side, Windows editor usually won't do this and you'll have to do it manually.